Src/BootCamp.Chapter/Lesson10.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson10
+    { 
+
+        const int CYPHERLENGTH = 95; // number of characters before it loops to the start
+        const int UNUSABLE = 32; //the first 32 chars of ascii are not printable
+   
+        public static string EncryptCypher(string message, int shift)
```

Comment:
``EncryptCypher`` is redundant. ``Encyrpt`` is enough.

Reply:
Removed EncryptCypher method and moved content straight to Encrypt